src/libraries/Directory.Build.targets

```diff
@@ -17,7 +17,9 @@
     <NetCoreAppCurrentTestHostPath>$([MSBuild]::NormalizeDirectory('$(ArtifactsBinDir)', 'testhost', '$(NetCoreAppCurrentBuildSettings)'))</NetCoreAppCurrentTestHostPath>
     <NetCoreAppCurrentTestHostSharedFrameworkPath>$([MSBuild]::NormalizeDirectory('$(NetCoreAppCurrentTestHostPath)', 'shared', '$(MicrosoftNetCoreAppFrameworkName)', '$(ProductVersion)'))</NetCoreAppCurrentTestHostSharedFrameworkPath>
     
-    <TargetsForTfmSpecificContentInPackage Condition="'$(IsPackable)' == 'true'">$(TargetsForTfmSpecificContentInPackage);LibIntellisenseDocs</TargetsForTfmSpecificContentInPackage>
+    <TargetsForTfmSpecificContentInPackage Condition="'$(IsPackable)' == 'true'">$(TargetsForTfmSpecificContentInPackage);AddRefAssemblies;AddRuntimeSpecificAssemblies;LibIntellisenseDocs</TargetsForTfmSpecificContentInPackage>
+    <TargetsForTfmSpecificDebugSymbolsInPackage Condition="'$(IsPackable)' == 'true'">$(TargetsForTfmSpecificDebugSymbolsInPackage);AddRefDebugSymbolsWithTfm;AddRuntimeDebugSymbolsWithTfm</TargetsForTfmSpecificDebugSymbolsInPackage>
+    <NoWarn Condition="'$(IsPackable)' == 'true'">$(NoWarn);NU5131</NoWarn>
```

Comment:
What is this warning?

Reply:
It warns about not having referenceGroups for the reference assemblies. we dont specify assembly references in our packages.

Reply:
```
error NU5131: References were found in the nuspec, but some reference assemblies were not found in both the nuspec and ref folder. Add the following reference assemblies: 
```

https://docs.microsoft.com/en-us/nuget/reference/nuspec